### PR TITLE
List Contacts UI

### LINKED ISSUE #4: Create list contacts UI

### PR DESCRIPTION
**Pull request by JaganKaartik**
Fixes #4

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            <th className="text-center">Contact Number</th>
+            <th className="text-center">Add to Basecamp</th>
+            <th className="text-center"></th>
+          </tr>
+        </thead>
+        <tbody>
+          {contacts.map(contact => (
+            <tr
+              key={contact.id}
+              className={"cursor-pointer bg-white hover:bg-gray-50"}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx:43]: Do we need these classnames?

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                  selectedContactIds.length ===
+                  contacts.map(contact => contact.id).length
+                }
+                onClick={() => {
+                  const contactIds = contacts.map(contact => contact.id);
+                  if (selectedContactIds.length === contactIds.length) {
+                    setSelectedContactIds([]);
+                  } else {
+                    setSelectedContactIds(contactIds);
+                  }
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx:27]: Please extract the logic to a separate function.

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                      setSelectedContactIds([
+                        ...selectedContactIds.slice(0, index),
+                        ...selectedContactIds.slice(index + 1),
+                      ]);
+                    } else {
+                      setSelectedContactIds([
+                        ...selectedContactIds,
+                        contact.id,
+                      ]);
+                    }
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx:62]: Please extract the logic to a separate function.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from "react";
+
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:2]: Please remove this line break here.

**Review by karthiknmenon — COMMENTED** [review]
@JaganKaartik Please check the suggestions.

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx (hunk 1 above)
Comment (JaganKaartik) [reply app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx:43]: ListContacts design had this style which is similar to NotesTable. Should I remove these classes or keep them?

Comment (karthiknmenon) [reply app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx:43]: You can remove the classes and use the default styles from neetoUI.

**Pull request merged**